index.js

```diff
@@ -32,11 +32,13 @@ Events.prototype.push = function(obj) {
 
 Events.prototype.flush = function() {
     if (!this.queue.length) return;
-    this._post(this.queue.splice(0, this.flushAt));
+    if (window.XDomainRequest) this._compatabilityPost(this.queue.splice(0, this.flushAt));
```

Comment:
`if (window.XDomainRequest)` will catch all IEs. The condition that [corslite](https://github.com/mapbox/corslite/blob/afb2c43f764ef637dca6585e3e96f9f6b64ddb77/corslite.js#L20) and [Mapbox.js](https://github.com/mapbox/mapbox.js/blob/fb502d6fd4cc0582fc4fadba7ba0d36b9a81ec4d/src/request.js#L13) use is:

```js
!('withCredentials' in new window.XMLHttpRequest())
```